EPlast/EPlast.Tests/Controllers/UserControllerTests.cs

```diff
@@ -0,0 +1,610 @@
+            Assert.IsInstanceOf<NotFoundResult>(result);
+        }
+
+        [Test]
+        public async Task ApproverDelete_ReturnsOkResult()
+        {
+            // Assert
+            var confirmedId = 1;
+
+            _confirmedUserService
+                .Setup((x) => x.DeleteAsync(It.IsAny<int>()));
+
+            // Act
+            var result = await _userController.ApproverDelete(confirmedId);
+
+            // Assert
+            _loggerService.Verify((x) => x.LogInformation(It.IsAny<string>()), Times.Once);
+            Assert.IsInstanceOf<OkResult>(result);
+        }
+
+        public UserDTO CreateFakeUser()
+            => new UserDTO()
+            {
+                Id = "1",
+                FirstName = "SomeFirstName",
+                LastName = "SomeLastName",
+                UserProfile = new UserProfileDTO()
+                {
+                    EducationId = 1,
+                    WorkId = 1,
+                },
+            };
+
+        public UserViewModel CreateFakeUserViewModel()
+            => new UserViewModel()
+            {
+                ID = "1",
+            };
+
+        public EditUserViewModel CreateFakeEditUserViewModel()
+            => new EditUserViewModel()
+            {
+                User = new UserViewModel()
+                {
+                    ID = "1",
+                },
+
+                ImageBase64 = "SomeImgInBase64",
+                
+                EducationView = new UserEducationViewModel()
+                {
+                    PlaceOfStudyID = 1,
+                    SpecialityID = 1,
+                },
+                
+                WorkView = new UserWorkViewModel()
+                {
+                    PlaceOfWorkID = 1,
+                    PositionID = 1,
+                },
+            };
+
+        public UserInfoViewModel CreateFaceUserInfoViewModel()
+            => new UserInfoViewModel()
+            {
+                Id = "1",
+            };
+
+        public List<ConfirmedUserViewModel> CreateListOfConfirmedUserViewModels()
+            => new List<ConfirmedUserViewModel>()
+            {
+                new ConfirmedUserViewModel()
+                {
+                    ID = 1,
+                    Approver = new ApproverViewModel()
+                    {
+                        User = new UserInfoViewModel()
+                        {
+                            ImagePath = "SomeStringID2",
+                        },
+                    },
+                },
+                new ConfirmedUserViewModel()
+                {
+                    ID = 2,
+                    Approver = new ApproverViewModel()
+                    {
+                        User = new UserInfoViewModel()
+                        {
+                            ImagePath = "SomeStringID1",
+                        },
+                    },
+                },
+            };
+
+        public ConfirmedUserViewModel CreateConfirmedUserViewModel()
```

Comment:
If you have faking function, use private modification.
```suggestion
        private ConfirmedUserViewModel CreateConfirmedUserViewModel()
```

Reply:
Done.